src/doc/html.md

```diff
@@ -199,7 +199,7 @@ world performance of your site and can show where you can optimize your specific
 site or application.
 
 
-### Google Universal Analytics Tracking Code
+### Asychronous Google Universal Analytics Tracking Code
```

Comment:
The official GA snippet has been asynchronous since way before Universal Analytics became a thing; the “Universal” in the title already implies that it’s asynchronous.